heatpump.yaml

```diff
@@ -335,6 +335,16 @@ sensor:
     name: Uptime
     id: "${devicename}_uptime"
     icon: mdi:timelapse
+  - platform: template
+    id: "${devicename}_cop"
+    name: "COP"
+    icon: mdi:copyleft
+    accuracy_decimals: 2
```

Comment:
Can you also add ` unit_of_measurement: cop`, this ensures that the value is show as a graph in Home Assistant instead of a history bar.

Reply:
did you checked that unknown for HA don't genmerate errors in HomeAssistant?

Reply:
OK. Good sugesstion ;) -I propose in uppercase as this is first letter shortcut from "Coefficient of Performance". This COP is behind value so -i think is better to change sensor name to full name to be better understand by others and short COP is still exists ;)
I've moved name before id -to continue schema of defining sensors ;)
It looks there isn't HA errors -so lets test it in this way ;)
What COP is in your pump? In warmer climate? ;) -in my area in winter temp sometimes drop to -20deg ;) -most of the time is +3 to -10 ;)